python/index_lambda.py

```diff
@@ -0,0 +1,129 @@
+#This code finds faces from an image, crops the faces, searches the faces in the AWS collection and adds them to a s3 bucket
+#if the image is not found in the collection
+
+import os
+import sys
+lib_path = os.path.abspath(os.path.join('lib'))
+sys.path.append(lib_path)
+import boto3
+from PIL import Image
+import botocore
+import io
+import urllib
+
+# Add SMS Subscribers
+list_of_contacts = ['+14049604358','+14704944634']
```

Comment:
This needs to be removed WE should not be committing items like this. We should be using SNS to send a message to a topic, not to send directly to a phone number, so we can build on top of the notification and then subscribe with whatever things we want.